Server_Side_TLS.mediawiki

```diff
@@ -251,26 +251,56 @@ As an example, TLS_DHE_RSA_WITH_AES_128_CBC_SHA256 works as follow:
 # Client sends a [http://tools.ietf.org/html/rfc5246#section-7.1 CHANGE CIPHER SPEC] message to the server, and both parties continue the handshake using ENCRYPTED HANDSHAKE MESSAGES
 
 The size of the prime number ''p'' constrains the size of the pre-master key ''PMS'', because of the modulo operation. A smaller prime almost means weaker values of ''A'' and ''B'', which could leak the secret values ''X'' and ''Y''. Thus, the prime ''p'' should not be smaller than the size of the RSA private key.
-<source lang="bash">
-$ openssl dhparam 2048
-Generating DH parameters, 2048 bit long safe prime, generator 2
-..+..+...............+
+
+== Pre-defined DHE groups ==
+
+Instead of using pre-configured DH groups, or generating their own with "openssl dhparam", operators should use the pre-defined DH groups ffdhe2048, ffdhe3072 or ffdhe4096 recommended by the IETF in [RFC 7919 https://tools.ietf.org/html/rfc7919].  These groups are audited and may be more resistant to attacks than ones randomly generated.
+
+Note: if you must support older Java clients, Dh groups larger than 1024 bits may block connectivity (see [[#DHE_and_Java]]).
```

Comment:
given that JRE 6 is only supported with a paid-for contract with Oracle, I think we can just call them "old", not "older" Java clients. Also the H in "Dh" should be upper case, it's an acronym.